pages/index.tsx

```diff
@@ -77,12 +73,12 @@ export const HomePage: React.FunctionComponent = () => {
                 We help you to verify the certificates you have of anyone from any government agencies. All in one
                 place.
               </p>
-              <Link to="/verify">
+              <Link href="/verify" passHref>
```

Comment:
```
passHref - Forces Link to send the href property to its child. Defaults to false
```

button doesn't have href property. Should be wrapped with a tag

Reply:
hm? sorry i'm not sure what you mean. I added the passHref because of `href - The path or URL to navigate to. This is the only required prop` in https://nextjs.org/docs/api-reference/next/link